integration_tests/integration/monitor.spec.js

```diff
@@ -257,6 +257,19 @@ describe('Probation Practitioner monitor journey', () => {
 
       cy.contains('Cancel referral').click()
       cy.contains('This referral has been cancelled')
+
+      const referralWithEndRequested = sentReferralFactory.endRequested().build({
+        ...referralParams,
+        assignedTo: { username: probationPractitioner.username },
+        actionPlanId: actionPlan.id,
+        endRequestedAt: '2021-04-04T20:45:21.986389Z',
```

Comment:
The endRequestedAt var is already defined in the factory. (I'm guessing it's to try and fix a broken intervention test)

Reply:
Yeah - I put this in (even though it's essentially a duplicate value to the factory) as I feel it's a bit clearer to have the test setup in the same place as the expectation for the date being displayed below, so you don't have to click into another file (the factory one) to see what date value we're testing here - does that feel like duplication / a bit over the top to you?

Reply:
Sorry, I must sound like I'm nitpicking; I assumed it was something you missed as I did something similar to work out why it wasn't working due to the date typo.

I slightly prefer having it within the factory class because there is a single point of reference to reduce work on any change to model structure; but it makes sense this way as well.